dapp/src/store/wallet/walletSelector.ts

```diff
@@ -13,6 +13,11 @@ export const selectActivities = createSelector(
   (activities) => sortActivitiesByTimestamp(activities),
 )
 
+export const selectHasPendingActivities = createSelector(
+  (state: RootState) => state.wallet.activities,
+  (activities) => activities.some((activity) => activity.status === "pending"),
+)
+
```

Comment:
I have confirmed the requirements and the tooltip should only show up when the user has pending deposits.

```suggestion
export const selectHasPendingDeposits = createSelector(
  (state: RootState) => state.wallet.activities,
  (activities) =>
    activities.some(
      (activity) =>
        activity.status === "pending" && activity.type === "deposit",
    ),
)
```